packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1566,6 +1607,36 @@ class EditableText extends StatefulWidget {
   /// {@macro flutter.services.TextInputConfiguration.enableIMEPersonalizedLearning}
   final bool enableIMEPersonalizedLearning;
 
+  /// {@template flutter.widgets.editableText.contentCommitMimeTypes}
+  /// Used when a user inserts image-based content through the device keyboard,
+  /// currently only used on Android.
+  ///
+  /// The passed list of strings will determine which MIME types are allowed to
+  /// be inserted via the device keyboard. If no `onContentCommitted` callback
+  /// is provided, this field will be ignored.
```

Comment:
What do you think about adding an assert to make sure that both contentCommitMimeTypes and onContentCommitted are passed if one is passed?

Reply:
Added